Charon/filetypes/UltimakerFormatPackage.py

```diff
@@ -257,6 +259,8 @@ def addRelation(self, virtual_path: str, relation_type: str, origin: str = "") -
     #   \return ``True`` if it exists as a normal resource, or ``False`` if it
     #   doesn't.
     def _resource_exists(self, virtual_path: str) -> bool:
```

Comment:
You didn't write it it, but should be _resourceExists

Reply:
All comments are fine, trying to cleanup the code a bit more, remember ;-)